keps/sig-auth/20190730-oidc-discovery.md

```diff
@@ -97,34 +130,38 @@ If services (other than the API server could authenticate KSA tokens directly:
     authenticate KSA tokens.
 -   Attempt compatibility with OIDC: common libraries that authenticate OIDC
     tokens should be able to authenticate KSA tokens.
-
-As a stretch goal / consideration:
-
 -   Support authentication when the API server is not directly reachable by the
     relying party.
     -   e.g.: a cloud-based service authenticating an API server that doesn't
```

Comment:
Is this saying that the relying party will be able to reach the KAS OIDC discovery document but not reach the rest of the API?

Reply:
Yes. For example, something with access to (or responsible for the deployment of) the cluster can cache the responses from the API server in a storage bucket, as long as either (a) the issuer URL is set to the bucket location or (b) the /.well-known path at the issuer URL redirects to the bucket.